pattoo_shared/agent.py

```diff
@@ -304,6 +333,16 @@ def __init__(self, parent, child, app, config=None):
             ip_bind_port=_config.ip_bind_port(),
             ip_listen_address=_config.ip_listen_address())
 
+        # Add email address to Agent subclass
+        econfig = Config()
+        email_addr = econfig.api_email_address()
+        self.set_email(email_addr)
```

Comment:
make _config self._config and you'll be able to get the email address available throughout the class.